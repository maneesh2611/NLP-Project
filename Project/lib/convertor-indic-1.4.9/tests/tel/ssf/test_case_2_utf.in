1	హైదరాబాద్	unk	<fs af='హైదరాబాద్,n,,sg,,d,0,0'>|<fs af='హైదరాబాద్,n,,sg,,o,టి,ti'>
2	లో	unk	<fs af='లో,avy,,,,,,'>|<fs af='ల,avy,,,,,ఓ,o_avy'>
3	గురు	unk	<fs af='గురువు,n,,sg,,,ఈ,I'>|<fs af='గురువు,n,,sg,,o,టి,ti'>
4	,	unk	<fs af=',,punc,,,,,'>
5	శుక్రవారాల్లో	unk	<fs af='శుక్రవారం,n,,pl,,,లో,lo'>
6	జస్టిస్	unk	<fs af='జస్టిస్,n,,sg,,d,0,0'>|<fs af='జస్టిస్,n,,sg,,o,టి,ti'>
7	శ్రీకృష్ణ	unk	<fs af='శ్రీకృష్ణ,n,,sg,,d,0,0'>|<fs af='శ్రీకృష్ణ,n,,sg,,o,టి,ti'>|<fs af='శ్రీకృష్ణుడు,n,,sg,,o,టి,ti'>
8	కమిటీ	unk	<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>
9	పర్యటన	unk	<fs af='పర్యటన,n,,sg,,d,0,0'>|<fs af='పర్యటన,n,,sg,,o,టి,ti'>|<fs af='పర్యటనం,n,,sg,,o,టి,ti'>
10	కేవలం	unk	<fs af='కేవలం,n,,sg,,d,0,0'>|<fs af='కేవ,n,,pl,,d,ను,nu'>
11	పరిచయానికి	unk	<fs af='పరిచయం,n,,sg,,,కి,ki'>
12	,	unk	<fs af=',,punc,,,,,'>
13	అవగాహన	unk	<fs af='అవగాహన,n,,sg,,d,0,0'>|<fs af='అవగాహన,n,,sg,,o,టి,ti'>|<fs af='అవగాహనం,n,,sg,,o,టి,ti'>
14	కోసం	unk	<fs af='కోసం,avy,,,,,0,0_avy'>
15	మాత్రమే	unk	<fs af='మాత్రమే,avy,,,,,,'>|<fs af='మాత్రం,avy,,,,,ఏ,e_avy'>
16	అని	unk	<fs af='అని,avy,,,,,0,0_avy'>|<fs af='అను,v,any,any,any,,ఇ,i'>
17	కమిటీ	unk	<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>
18	సభ్య	unk	<fs af='సభ్యం,n,,sg,,o,టి,ti'>|<fs af='సభ్యుడు,n,,sg,,o,టి,ti'>
19	కార్యదర్శి	unk	<fs af='కార్యదర్శి,n,,sg,,d,0,0'>|<fs af='కార్యదర్శి,n,,sg,,o,టి,ti'>
20	వినోద్	unk	<fs af='వినోద్,unk,,,,,,'>
21	కె	unk	<fs af='కె,avy,,,,,0,0_avy'>
22	దుగ్గల్	unk	<fs af='దుగ్గల్,unk,,,,,,'>
23	తెలిపారు	unk	<fs af='తెలుపు,v,fm,pl,3,,ఆ,A'>
24	।	unk	<fs af='।,punc,,,,,,'>
