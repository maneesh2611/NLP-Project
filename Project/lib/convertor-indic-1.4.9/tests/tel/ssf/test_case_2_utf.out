1	hExarAbAx	unk	<fs af='hExarAbAx,n,,sg,,d,0,0'>|<fs af='hExarAbAx,n,,sg,,o,ti,ti'>
2	lo	unk	<fs af='lo,avy,,,,,,'>|<fs af='la,avy,,,,,o,o_avy'>
3	guru	unk	<fs af='guruvu,n,,sg,,,I,I'>|<fs af='guruvu,n,,sg,,o,ti,ti'>
4	,	unk	<fs af=',,punc,,,,,'>
5	SukravArAllo	unk	<fs af='SukravAraM,n,,pl,,,lo,lo'>
6	jastis	unk	<fs af='jastis,n,,sg,,d,0,0'>|<fs af='jastis,n,,sg,,o,ti,ti'>
7	SrIkqRNa	unk	<fs af='SrIkqRNa,n,,sg,,d,0,0'>|<fs af='SrIkqRNa,n,,sg,,o,ti,ti'>|<fs af='SrIkqRNudu,n,,sg,,o,ti,ti'>
8	kamitI	unk	<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,o,ti,ti'>|<fs af='kamitI,n,,sg,,o,ti,ti'>
9	paryatana	unk	<fs af='paryatana,n,,sg,,d,0,0'>|<fs af='paryatana,n,,sg,,o,ti,ti'>|<fs af='paryatanaM,n,,sg,,o,ti,ti'>
10	kevalaM	unk	<fs af='kevalaM,n,,sg,,d,0,0'>|<fs af='keva,n,,pl,,d,nu,nu'>
11	paricayAniki	unk	<fs af='paricayaM,n,,sg,,,ki,ki'>
12	,	unk	<fs af=',,punc,,,,,'>
13	avagAhana	unk	<fs af='avagAhana,n,,sg,,d,0,0'>|<fs af='avagAhana,n,,sg,,o,ti,ti'>|<fs af='avagAhanaM,n,,sg,,o,ti,ti'>
14	kosaM	unk	<fs af='kosaM,avy,,,,,0,0_avy'>
15	mAwrame	unk	<fs af='mAwrame,avy,,,,,,'>|<fs af='mAwraM,avy,,,,,e,e_avy'>
16	ani	unk	<fs af='ani,avy,,,,,0,0_avy'>|<fs af='anu,v,any,any,any,,i,i'>
17	kamitI	unk	<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,o,ti,ti'>|<fs af='kamitI,n,,sg,,o,ti,ti'>
18	saBya	unk	<fs af='saByaM,n,,sg,,o,ti,ti'>|<fs af='saByudu,n,,sg,,o,ti,ti'>
19	kAryaxarSi	unk	<fs af='kAryaxarSi,n,,sg,,d,0,0'>|<fs af='kAryaxarSi,n,,sg,,o,ti,ti'>
20	vinox	unk	<fs af='vinox,unk,,,,,,'>
21	keV	unk	<fs af='keV,avy,,,,,0,0_avy'>
22	xuggal	unk	<fs af='xuggal,unk,,,,,,'>
23	weVlipAru	unk	<fs af='weVlupu,v,fm,pl,3,,A,A'>
24	ред	unk	<fs af='ред,punc,,,,,,'>
